.github/workflows/sycl_nightly.yml

```
@@ -54,4 +54,12 @@ jobs:
         tags: |
           ghcr.io/${{ github.repository }}/sycl_ubuntu2004_nightly:no-drivers-${{ github.sha }}
           ghcr.io/${{ github.repository }}/sycl_ubuntu2004_nightly:no-drivers
-
+  ubuntu2004_build_test_new_pm:
+    # Default Linux building and LIT testing using new Pass Manager by default
+    if: github.repository == 'intel/llvm'
+    uses: intel/llvm/.github/workflows/sycl_linux_build_and_test.yml@sycl
+    with:
+      build_runs_on: build
+      build_cache_root: "/__w/"
+      build_artifact_suffix: new_pm
+      build_configure_extra_args: '--hip --hip-amd-arch=gfx906 --cuda --cmake-opt="-DLLVM_ENABLE_NEW_PASS_MANAGER=ON"'
```

Based on our offline discussion.

```suggestion
      build_configure_extra_args: '--hip --hip-amd-arch=gfx906 --cuda --cmake-opt="-DLLVM_ENABLE_NEW_PASS_MANAGER=ON"'
      lts_config: "hip_amdgpu;ocl_x64"
```